app/page.tsx

```diff
@@ -76,6 +77,21 @@ export default function Home() {
               <p className="text-center text-3xl font-bold">
                 {data?.[currentIndex]?.answer}
               </p>
+              <Button
+                className="absolute bottom-5 right-5 text-zinc-500"
+                variant="link"
+                asChild
+              >
+                <Link
+                  onClick={(e) => e.stopPropagation()}
+                  href={{
+                    pathname: "/chat",
+                    query: { answer: data?.[currentIndex]?.answer },
```

Comment:
we can use the id of the card later instead of any text here. anyone can configure the system prompt here in the url which is not ideal